pkg/specgenutil/specgen.go

```diff
@@ -865,6 +867,9 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *entities.ContainerCreateOptions
 			logOpts[key] = val
 		}
 	}
+	if !useLogOpts && rtc.Containers.LogPath != "" {
+		s.LogConfiguration.Path = rtc.Containers.LogPath
+	}
```

Comment:
same thing here, although I Really wonder why we have the same parsing loop duplicated twice with slightly different options parsing. Maybe no something you have to fix but this looks like something to me that should really be de-duplicated